Sources/App/Models/User/User.swift

```diff
@@ -35,31 +36,67 @@ final class User: Model {
   init(row: Row) throws {
     name = try row.get(Keys.name)
     lastname = try row.get(Keys.lastname)
-    company = try row.get(Keys.company)
-    position = try row.get(Keys.position)
-    photo = try row.get(Keys.photo)
-    email = try row.get(Keys.email)
-    phone = try row.get(Keys.phone)
+    company = try? row.get(Keys.company)
+    position = try? row.get(Keys.position)
+    photo = try? row.get(Keys.photo)
+    email = try? row.get(Keys.email)
+    phone = try? row.get(Keys.phone)
   }
 
   func makeRow() throws -> Row {
     var row = Row()
     try row.set(Keys.name, name)
     try row.set(Keys.lastname, lastname)
-    try row.set(Keys.company, company)
-    try row.set(Keys.position, position)
-    try row.set(Keys.photo, photo)
-    try row.set(Keys.email, email)
-    try row.set(Keys.phone, phone)
+    try? row.set(Keys.company, company)
+    try? row.set(Keys.position, position)
+    try? row.set(Keys.photo, photo)
+    try? row.set(Keys.email, email)
+    try? row.set(Keys.phone, phone)
     return row
   }
   // sourcery:end
 }
 
+extension User: JSONRepresentable {
+  
+  func makeJSON() throws -> JSON {
+    var json = JSON()
+    try json.set(Keys.id, id)
+    try json.set(Keys.name, name)
+    try json.set(Keys.lastname, lastname)
+    try? json.set(Keys.company, company)
+    try? json.set(Keys.position, position)
+    try? json.set(Keys.photo, photo)
+    try? json.set(Keys.email, email)
+    try? json.set(Keys.phone, phone)
+    try json.set("session", session()?.makeJSON())
```

Comment:
Why code generation didn't work here? It seems no custom stuff here

Reply:
I've added it because of additional `session` key. But not it has custom `photo` value. So left it as is.

Reply:
I got you right, you have only session key that is custom here?
have you tried `// sourcery: nestedJSONRepresentableField` like in `Place` model? 
it will generate session key as well